projects/go-lib/src/lib/components/go-action-sheet/go-panel/go-panel.component.html

```diff
@@ -1,6 +1,7 @@
 <!-- External Link -->
 <a
   class="go-panel"
+  [class.go-panel__disabled]="disablePanel"
```

Comment:
Should we use the `ngClass` on line 7 instead of this?

Reply:
Like your approach more. It is cleaner!